apps/web/src/hooks/use-database.ts

```diff
@@ -28,14 +28,14 @@ if (process.env.NODE_ENV === "production") {
 const memory = {
   isAppLoaded: false
 };
-export default function useDatabase(persistence) {
+export default function useDatabase(persistence?: "db" | "memory") {
   const [isAppLoaded, setIsAppLoaded] = useState(memory.isAppLoaded);
 
   useEffect(() => {
     if (memory.isAppLoaded) return;
 
     (async () => {
-      await import("../app.css");
+      await import("../app.css" as string);
```

Comment:
Why is this needed?

Reply:
This problem I faced and found out Typescript won't let us import the css file it will only allowed .ts file that is why as string is necessary to import the .css file.

Reply:
Otherwise there was an error module not found.